src/renderers/dom/fiber/ReactDOMFiberEntry.js

```diff
@@ -757,8 +758,27 @@ function createPortal(
   return ReactPortal.createPortal(children, container, null, key);
 }
 
+type WorkNode = {
+  commit(): void,
+
+  _reactRootContainer: *,
```

Comment:
Why do we use an existential type here instead of something explicit?

Reply:
Because I was lazy and didn't want to import it :D

Reply:
:D